pkg/path/path.go

```diff
@@ -36,3 +39,11 @@ func GetOsData() string {
 	goVersion := strings.ReplaceAll(runtime.Version(), "go", "")
 	return "[OS: " + strings.ToUpper(os) + "] [ARCH: " + strings.ToUpper(arc) + "] [GO: " + goVersion + "]"
 }
+
+func GetExecutablePath(path, toolName string) string {
```

Comment:
I think here it can be refactored as (pseudocode):
```go
commonExtensions := []string{"", ".exe", ".bat", ...}

func GetExecutablePath(path, toolName string) (string,bool) {
     basePath := filepath.Join(path, toolName)
     for _, ext := range commonExtensions {
        executablePath :=  basePath + ext
	if fileutil.FileExists(executablePath) {
	        return executablePath, true
	}
     }
     
     if runtime.GOOS == "windows" {
          return basePath + ".exe", false
     }
     
     return basePath, false
}
```

The function should account for `.exe` on windows also in case the file doesn't exist